src/Serializers/Native.php

```diff
@@ -140,6 +140,11 @@ public function __serialize()
 
         $code = $reflector->getCode();
 
+
+        if (PHP_VERSION >= 8.4) {
+            $code = str_replace('\{closure:', '', $code);
+        }
+
```

Comment:
This changes fixes the problem...

See Runs in https://github.com/laravel/framework/pull/52919